addons/l10n_it_edi_sdicoop/models/res_config_settings.py

```diff
@@ -0,0 +1,25 @@
+# -*- coding: utf-8 -*-
+
+from odoo import api, models, fields
+
+
+class ResConfigSettings(models.TransientModel):
+    _inherit = 'res.config.settings'
+
+    is_edi_proxy_active = fields.Boolean(compute='_compute_is_edi_proxy_active')
+
+    @api.depends('company_id.account_edi_proxy_client_ids', 'company_id.account_edi_proxy_client_ids.active')
+    def _compute_is_edi_proxy_active(self):
+        for config in self:
+            config.is_edi_proxy_active = config.company_id.account_edi_proxy_client_ids
+
+    def button_create_proxy_user(self):
+        # For now, only fattura_pa uses the proxy.
+        # To use it for more, we have to either make the activation of the proxy on a format basis
+        # or create a user per format here (but also when installing new formats)
```

Comment:
not sure there is still something to do here.

Reply:
What do you mean ? See my comment here https://github.com/odoo/odoo/pull/67923#discussion_r626296708 If we want to support more formats, we will need to make some changes here, but I think it's premature to think of that now, that's why this feature is in l10n_it_edi_sdicoop and not in account_edi_proxy_client.